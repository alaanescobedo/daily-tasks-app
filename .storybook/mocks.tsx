import { Time } from '../src/components'

const runMocks = () => {
  jest.mock('components/Form/Time/Time.tsx', () => ({
    Time: () => <Time defaultValue='12:00' id='time' handleChange={() => { }} />,
  }));
}

export default runMocks