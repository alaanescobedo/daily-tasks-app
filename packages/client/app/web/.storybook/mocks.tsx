import { TimeField } from "@ui/components";

const runMocks = () => {
  jest.mock('components/Form/Time/Time.tsx', () => ({
    Time: () => <TimeField defaultValue='12:00' id='time' handleChange={() => { }} />,
  }));
}

export default runMocks