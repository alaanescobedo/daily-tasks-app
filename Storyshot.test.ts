import initStoryshots from '@storybook/addon-storyshots'
import './.storybook/mocks'

initStoryshots()
