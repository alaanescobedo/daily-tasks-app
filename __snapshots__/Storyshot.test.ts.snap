// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button Add Task 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <a
      className="btn"
      href="/new-task"
      onClick={[Function]}
    >
      <p
        className="label"
      >
        Add Task
      </p>
    </a>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/Auth Email 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <div
      className="form_group"
    >
      <label
        htmlFor="email"
      >
        Email
      </label>
      <input
        autoComplete="off"
        id="email"
        type="email"
      />
    </div>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/Checkbox Input Checkbox 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <span
      className="input_recurrent"
      onClick={[Function]}
    >
      <svg
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"
        />
      </svg>
      <input
        checked={true}
        className="checkbox"
        id="recurrent"
        name="recurrent"
        readOnly={true}
        type="checkbox"
      />
    </span>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/InputGroup Group Checkbox 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <div
      className="group_recurrent"
    >
      <span
        className="input_recurrent"
        onClick={[Function]}
      >
        <svg
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"
          />
        </svg>
        <input
          checked={true}
          className="checkbox"
          id="recurrent"
          name="recurrent"
          readOnly={true}
          type="checkbox"
        />
      </span>
      <label
        htmlFor="recurrent"
      >
        Recurrent
      </label>
    </div>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/InputGroup Group Date 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <div
      className="group_date_day"
    >
      <select
        className="input_select"
        defaultValue="friday"
        id="date"
        name="date"
        onChange={[Function]}
      >
        <option
          value="friday"
        >
          Friday
        </option>
        <option
          value="saturday"
        >
          Saturday
        </option>
        <option
          value="sunday"
        >
          Sunday
        </option>
        <option
          value="monday"
        >
          Monday
        </option>
        <option
          value="tuesday"
        >
          Tuesday
        </option>
        <option
          value="wednesday"
        >
          Wednesday
        </option>
        <option
          value="thursday"
        >
          Thursday
        </option>
      </select>
      <label
        htmlFor="date"
      >
        Date
      </label>
    </div>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/InputGroup Group Hour 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <div
      className="group_date_hour"
    >
      <input
        className="input_date_hour"
        defaultValue="12:00"
        id="time"
        min="12:00"
        name="time"
        onChange={[Function]}
        type="time"
      />
      <label
        htmlFor="time"
      >
        Hour
      </label>
    </div>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/InputGroup Group Priority 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <div
      className="group_priority"
    >
      <select
        className="input_select"
        defaultValue="medium"
        id="priority"
        name="priority"
        onChange={[Function]}
      >
        <option
          value="low"
        >
          Low
        </option>
        <option
          value="medium"
        >
          Medium
        </option>
        <option
          value="high"
        >
          High
        </option>
      </select>
      <label
        htmlFor="priority"
      >
        Priority
      </label>
    </div>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/Select Date Select 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <select
      className="input_select"
      defaultValue="friday"
      id="date"
      name="date"
      onChange={[Function]}
    >
      <option
        value="friday"
      >
        Friday
      </option>
      <option
        value="saturday"
      >
        Saturday
      </option>
      <option
        value="sunday"
      >
        Sunday
      </option>
      <option
        value="monday"
      >
        Monday
      </option>
      <option
        value="tuesday"
      >
        Tuesday
      </option>
      <option
        value="wednesday"
      >
        Wednesday
      </option>
      <option
        value="thursday"
      >
        Thursday
      </option>
    </select>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/Select Priority Select 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <select
      className="input_select"
      defaultValue="medium"
      id="priority"
      name="priority"
      onChange={[Function]}
    >
      <option
        value="low"
      >
        Low
      </option>
      <option
        value="medium"
      >
        Medium
      </option>
      <option
        value="high"
      >
        High
      </option>
    </select>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/Textarea Primary 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <textarea
      className="input_task"
      id="textarea"
      maxLength={70}
      name="textarea"
      onChange={[Function]}
      placeholder="Title"
      value=""
    />
     
  </div>,
  " ",
]
`;

exports[`Storyshots Form/Time Input Time 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <input
      className="input_date_hour"
      defaultValue="12:00"
      id="time"
      min="12:00"
      name="time"
      onChange={[Function]}
      type="time"
    />
     
  </div>,
  " ",
]
`;

exports[`Storyshots Layout/AppContainer App Container 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <span
      className="app_container"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <div
          className="container"
        >
          <h4
            className="username"
          >
            Username
          </h4>
          <h4
            className="date"
          >
            Tuesday,  4/12/2022
            <div
              style={
                Object {
                  "display": "none",
                }
              }
            >
              Clock
            </div>
          </h4>
        </div>
      </a>
      <div
        className="super_container"
      >
        <div
          className="container"
        >
          <span
            style={
              Object {
                "color": "white",
                "flex": "1",
              }
            }
          >
            ʕ•́ᴥ•̀ʔっayo!
          </span>
        </div>
        <div
          className="return_container"
        >
          <a
            aria-current="page"
            className="return active"
            href="/"
            onClick={[Function]}
          >
            Return
          </a>
        </div>
      </div>
    </span>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Layout/AuthLayout Auth Layout 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <span
      className="app_container"
    >
      <div
        className="container"
      >
        <div
          className="container_heading"
        >
          <h1
            className="heading"
          >
            AuthLayout
          </h1>
        </div>
        <form
          className="form"
          onSubmit={[Function]}
        >
          <span
            style={
              Object {
                "color": "white",
                "flex": "1",
              }
            }
          >
            ʕ•́ᴥ•̀ʔっayo!
          </span>
          <div
            className="btn_container"
          >
            <button
              className="btn_continue"
            >
              Continue
            </button>
          </div>
        </form>
      </div>
    </span>
     
  </div>,
  " ",
]
`;

exports[`Storyshots List/Activities List Activities 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <ul
      className="container"
    >
      <span
        style={
          Object {
            "display": "flex",
            "gap": "1rem",
          }
        }
      >
        <li
          className="list_item"
        >
          Task 1
        </li>
        <span
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
          Remove
        </span>
        <a
          href="/activities/update/2022-04-11"
          onClick={[Function]}
        >
          Update
        </a>
      </span>
    </ul>
     
  </div>,
  " ",
]
`;

exports[`Storyshots List/Tasks List Of Tasks 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <ul
      className="container"
    >
      <a
        className=""
        href="/activities/2022-04-11"
        onClick={[Function]}
      >
        <div
          className="container"
        >
          <ul
            className="list"
          >
            <li>
              - 
              Task 1
            </li>
          </ul>
          <div
            className="day"
          >
            <p>
              2022-04-11
            </p>
          </div>
        </div>
      </a>
      <a
        className=""
        href="/activities/2022-04-12"
        onClick={[Function]}
      >
        <div
          className="container"
        >
          <ul
            className="list"
          >
            <li>
              - 
              Task 2
            </li>
            <li>
              - 
              Task 3
            </li>
          </ul>
          <div
            className="day"
          >
            <p>
              2022-04-12
            </p>
          </div>
        </div>
      </a>
      <a
        className=""
        href="/activities/2022-04-13"
        onClick={[Function]}
      >
        <div
          className="container"
        >
          <ul
            className="list"
          >
            <li>
              - 
              Task 4
            </li>
          </ul>
          <div
            className="day"
          >
            <p>
              2022-04-13
            </p>
          </div>
        </div>
      </a>
      <a
        className=""
        href="/activities/2022-04-14"
        onClick={[Function]}
      >
        <div
          className="container"
        >
          <ul
            className="list"
          >
            <li>
              - 
              Task 5
            </li>
          </ul>
          <div
            className="day"
          >
            <p>
              2022-04-14
            </p>
          </div>
        </div>
      </a>
      <a
        className=""
        href="/activities/2022-04-15"
        onClick={[Function]}
      >
        <div
          className="container"
        >
          <ul
            className="list"
          >
            <li>
              - 
              Task 6
            </li>
          </ul>
          <div
            className="day"
          >
            <p>
              2022-04-15
            </p>
          </div>
        </div>
      </a>
    </ul>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Scroll/Scroll Scroll 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
    <div
      className="container"
    >
      <ul
        className="container"
      >
        <a
          className=""
          href="/activities/2022-04-11"
          onClick={[Function]}
        >
          <div
            className="container"
          >
            <ul
              className="list"
            >
              <li>
                - 
                Task 1
              </li>
            </ul>
            <div
              className="day"
            >
              <p>
                2022-04-11
              </p>
            </div>
          </div>
        </a>
        <a
          className=""
          href="/activities/2022-04-12"
          onClick={[Function]}
        >
          <div
            className="container"
          >
            <ul
              className="list"
            >
              <li>
                - 
                Task 2
              </li>
              <li>
                - 
                Task 3
              </li>
            </ul>
            <div
              className="day"
            >
              <p>
                2022-04-12
              </p>
            </div>
          </div>
        </a>
        <a
          className=""
          href="/activities/2022-04-13"
          onClick={[Function]}
        >
          <div
            className="container"
          >
            <ul
              className="list"
            >
              <li>
                - 
                Task 4
              </li>
            </ul>
            <div
              className="day"
            >
              <p>
                2022-04-13
              </p>
            </div>
          </div>
        </a>
        <a
          className=""
          href="/activities/2022-04-14"
          onClick={[Function]}
        >
          <div
            className="container"
          >
            <ul
              className="list"
            >
              <li>
                - 
                Task 5
              </li>
            </ul>
            <div
              className="day"
            >
              <p>
                2022-04-14
              </p>
            </div>
          </div>
        </a>
        <a
          className=""
          href="/activities/2022-04-15"
          onClick={[Function]}
        >
          <div
            className="container"
          >
            <ul
              className="list"
            >
              <li>
                - 
                Task 6
              </li>
            </ul>
            <div
              className="day"
            >
              <p>
                2022-04-15
              </p>
            </div>
          </div>
        </a>
      </ul>
    </div>
     
  </div>,
  " ",
]
`;

exports[`Storyshots Views/Activities Activities 1`] = `
Array [
  " ",
  <div
    style={
      Object {
        "--background": "2, 40, 60",
        "--highlight": "255,  202,  16",
        "--opacity-1": 0.1,
        "--opacity-7": 0.7,
        "--primary": "206, 180, 249",
        "--secondary": "189, 231, 255",
        "--tertiary": "176, 239, 165",
        "--text": "245, 245, 245",
        "0.1": "--opacity-1",
        "0.7": "--opacity-7",
      }
    }
  >
     
     
  </div>,
  " ",
]
`;
